### PR TITLE
juju 2.8.6

### PR DESCRIPTION
**Pull request by zachauten**
Created with `brew bump-formula-pr`.

### CODE REVIEW

File: Formula/juju.rb [hunk 1]
```diff
@@ -2,8 +2,8 @@ class Juju < Formula
   desc "DevOps management tool"
   homepage "https://juju.is/"
   url "https://github.com/juju/juju.git",
-    tag:      "juju-2.8.5",
-    revision: "1f35f6a20b57aefb350f04ae74e83ace32c73094"
+    tag:      "juju-2.8.6",
+    revision: "5d0442d3e15952bfc0ce059cb43ef7949ca71aaa"
   license "AGPL-3.0"
```
Comment (fxcoudert) [inline Formula/juju.rb:7]:
```suggestion
  license "AGPL-3.0-only"
```

https://github.com/juju/juju/blob/develop/juju/home.go
```
// Licensed under the AGPLv3, see LICENCE file for details.
```

**Review by fxcoudert — CHANGES_REQUESTED** [review]

File: Formula/juju.rb (hunk 1 above)
Comment (chenrui333) [reply Formula/juju.rb:7]: cc @wallyworld  @SimonRichardson, can you folks help confirming the license? APLv3 is a deprecated license, so we need to update it.

Comment (SimonRichardson) [reply Formula/juju.rb:7]: Yes the license is GNU Affero General Public License v3.0, AGPLv3.

https://github.com/juju/juju/blob/develop/LICENCE

**Review by fxcoudert — APPROVED** [review]

File: Formula/juju.rb (hunk 1 above)
Comment (chenrui333) [reply Formula/juju.rb]: Thank you!!

**Review by chenrui333 — APPROVED** [review]

**Pull request closed without merging**